atproto/xrpc_client/models/com/atproto/admin/take_moderation_action.py

```diff
@@ -32,3 +32,19 @@ class Data(base.DataModelBase):
     )  #: Indicates how long this action was meant to be in effect before automatically expiring.
     negate_label_vals: t.Optional[t.List[str]] = Field(default=None, alias='negateLabelVals')  #: Negate label vals.
     subject_blob_cids: t.Optional[t.List[str]] = Field(default=None, alias='subjectBlobCids')  #: Subject blob cids.
+
+
+class DataDict(te.TypedDict):
+    action: str  #: Action.
+    created_by: str  #: Created by.
+    reason: str  #: Reason.
+    subject: te.Annotated[
+        t.Union['models.ComAtprotoAdminDefs.RepoRef', 'models.ComAtprotoRepoStrongRef.Main'],
+        Field(discriminator='py_type'),
+    ]  #: Subject.
```

Comment:
is pydantic part in TypedDict fine?

Reply:
[the docs](https://docs.python.org/3/library/typing.html#typing.Annotated) suggest that it should simply be ignored but I can double check it later

Reply:
Both mypy and pyright are happy with it being there 👍